src/AccessibilityHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Tomaj\Form\Renderer;
+
+use Nette\Forms\Control;
+use Nette\Forms\Form;
+use Nette\Utils\Html;
+
+class AccessibilityHelper
```

Comment:
separated adding aria attributes to its own class - so we dont have duplicate code